### DISCUSSION TITLE
Does copier supports Conditional prompts?

### DISCUSSION THREAD

**Discussion by dragneelfps** [copier-org/copier#427] [Q&A] [answered]
I have a usecase where based on prompt A, I might have to enable/disable prompt B. 
For example, if answer to prompt A is "Enable multiplatform", only then I want user to be prompted with prompt B.
Does copier supports this?

**Comment by yajo** [comment, marked as answer]
Yes, if you install from git master and use the `when` key.

Check master docs: https://copier.readthedocs.io/en/latest/configuring/#questions
